backend/corpora/common/utils/dl_sources/url.py

```diff
@@ -72,8 +82,10 @@ def file_info(self) -> dict:
         resp = requests.head(self.url, allow_redirects=True)
         resp.raise_for_status()
 
+        print(resp.headers)
+
         return {
-            "size": int(self._get_key(resp.headers, "content-length")),
+            "size": int(self._get_key_with_fallback(resp.headers, "content-length", "x-dropbox-content-length")),
```

Comment:
where did you find `x-dropbox-content-length`?

Reply:
I created a notebook, replicated the call and inspected the headers. ```curl --head --http1.1 -L "https://www.dropbox.com/s/j0s89tokxwz3h2p/2_0_0_raw_valid.h5ad/?dl=1"``` can also be used.